server/game/cards/05-DT/GrandAllianceCouncil.js

```diff
@@ -0,0 +1,34 @@
+const Card = require('../../Card.js');
+const Houses = require('../../../constants').Houses;
+const HousesNames = require('../../../constants').HousesNames;
+
+class GrandAllianceCouncil extends Card {
+    //Play: Choose a creature of each house. Destroy each creature not chosen.
+    setupCardAbilities(ability) {
+        const targets = {};
+        for (const house of Houses) {
+            targets[house] = {
+                activePromptTitle: {
+                    text: 'Choose a {{house}} creature to not destroy',
+                    values: { house: HousesNames[Houses.indexOf(house)] }
+                },
+                cardType: 'creature',
+                numCards: 1,
+                cardCondition: (card) => card.hasHouse(house),
+                gameAction: ability.actions.destroy((context) => ({
+                    target: context.game.creaturesInPlay.filter(
+                        (card) => context.targets[house] !== card && card.hasHouse(house)
```

Comment:
there might be a corner case here where a creature belong to two houses. You need to check if the card is not selected at all. Also, you might want to check if targets is defined just in case, since the gameAction is outside target brackets:
target: context.targets ? context.game.creaturesInPlay.filter(
                        (card) => card.hasHouse(house) && !Object.values(context.targets).includes(card)) : []

Reply:
This is a minor issue, I will merge